### ISSUE TITLE
Potential bug in CaptainHook::Message.InjectIssueKeyFromBranch

### ISSUE THREAD

**Issue by tommy-muehle** [captainhook-go/captainhook#1]
Hi @sebastianfeldmann 👋 

not seen or heard for a long time! But I hope you're doing well. 🙂 
I'm currently playing around with Captain Hook for Go. Therefore I added the following configuration to my repository:

```json
"prepare-commit-msg": {
  "actions": [
     {
	"run": "CaptainHook::Message.InjectIssueKeyFromBranch",
	"options": {
	  "regex": "[A-Z]+-[0-9]+",
          "into": "subject",
	  "force": true
	}
     }
  ]
}
```

My branch name is just `PRODUCT-2453` and it's catched successfully! 

But I get a panic `panic: runtime error: index out of range [1] with length 1`.

If I looked correct and it's not too late it's because you're looking into `match[1]` instead of `match[0]` [here](https://github.com/captainhook-go/captainhook/blob/main/hooks/actions/message/injectissuekeyfrombranch.go#L67).

Proof of concept:
```go
package main

func main() {
  log.Println(regexp.MustCompile("[A-Z]+-[0-9]+").FindStringSubmatch("PRODUCT-2053")[0])
}
```

Best regards,
Tommy

**Comment by sebastianfeldmann** [comment]
Yes the, I have to make sure there is an index on.
I will patch this and will give a helpful warning.
The regex should contain a group like this `"([A-Z]+-[0-9]+)"` then ist should work.

I will go and fix the panic ;)

**Comment by tommy-muehle** [comment]
@sebastianfeldmann 
Can confirm, a group also fixes the issue!

**Comment by sebastianfeldmann** [comment]
Just released `1.0.2` it checks the match slice before using match[1]
If it is to short you get a message that you should pur your ISSUE-KEY in a regex group.

Thanks for the help :)
